### PR TITLE
[4/5] Replace all Golang + UI references to integration

### PR DESCRIPTION
**Pull request by kenxu95**
## Describe your changes and why you are making these changes
Turns out its a lot faster to simply rename everything all at once, and then check that nothing is messed up. For this PR, we do a dumb replace "integration" and "Integration" in all the code and filenames in src/golang and src/ui.

What is remaining
[5/5] The migration of all python code. This code is a little trickier to migrate since "integration tests" is a valid phrase.

## Related issue number (if any)
ENG-2780

## Checklist before requesting a review
- [ ] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [ ] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

**Comment by likawind** [comment]
I don't think we need to rename historical data versions for migrator? History stays in history :)

**Review by likawind — APPROVED** [review]

File: src/golang/lib/models/shared/engine.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ type AqueductCondaConfig struct {
 }
 
 type AirflowConfig struct {
-	ResourceID uuid.UUID `json:"integration_id"  yaml:"integration_id"`
```
Comment (saurav-c) [inline src/golang/lib/models/shared/engine.go:40]: Is this migration going to be taken care of as part of PR 6?

Comment (kenxu95) [reply src/golang/lib/models/shared/engine.go:40]: yep!

Comment (kenxu95) [reply src/golang/lib/models/shared/engine.go:40]: @saurav-c actually I think we're gonna give up on renaming the fields themselves due to time constraints. Synced with @likawind and it's actually quite a lot of work with low benefit, so I'll just make a follow up task.

**Pull request merged**